packages/reference/src/common/useEditorPermissions.spec.ts

```diff
@@ -63,10 +63,11 @@ describe('useEditorPermissions', () => {
       expect(sdk.access.can).not.toHaveBeenCalledWith();
     });
 
-    it(`defaults link entry action visibility to true`, async () => {
-      const { result } = await renderEditorPermissions({ entityType: 'Entry' });
+    it(`checks basic access`, async () => {
```

Comment:
I don't get the intention behind that change, can you briefly explain?

Reply:
I am reverting the changes done in this [pr](https://github.com/contentful/field-editors/pull/580) since it was not solving our problem